src/lib/libsdl.js

```diff
@@ -3122,7 +3122,11 @@ var LibrarySDL = {
       // valid. Check that explicitly and fall back to a plain Canvas if we need
       // to. See https://github.com/emscripten-core/emscripten/issues/16242
       if (typeof SDL.ttfContext.measureText != 'function') {
```

Comment:
Probably enough to just do `if (!SDL.ttfContext.measureText)`.. but again doesn't have to part of this PR.